0000-evidence-template.md

```diff
@@ -33,6 +34,20 @@
 |   |   |   |   |
 
 
+## Voting record
+*Provide your voting record in relation to required thresholds.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Links | Comments |
+|---|---|---|---|---|
+|III|70%   |100%  |[My votes](https://collectives.subsquare.io/user/13aYUFHB3umoPoxBEAHSv451iR3RpsNi3t5yBZjX2trCtTp6/votes) |*This is an example.* I have *[Never/Sometimes/Often/Always]* voted across all referendum tracks AND *[Never/Sometimes/Often/Always]* voted in line with expectations at rank. |
+|I  |90%   |N/A   |   |  |
+|II |80%   |N/A   |   |  |
+|III|70%   |100%  |   |  |
```

Comment:
100% thresholds doesn't make sense espically when we don't have abstain option. People should only vote if they understand fully about the proposal and it is not feasible to require rank 3 members to accurately understand all the technical details of all the proposals